extensions/grpc/protoc/src/main/java/io/quarkus/grpc/protoc/plugin/MutinyGrpcGenerator.java

```diff
@@ -41,9 +41,25 @@ private String getMethodJavaDocPrefix() {
                 .collect(Collectors.toList());
 
         List<ServiceContext> services = findServices(protosToGenerate, typeMap);
+        validateServices(services);
         return generateFiles(services);
     }
 
+    private void validateServices(List<ServiceContext> services) {
+        boolean failed = false;
+        for (ServiceContext service : services) {
+            if (service.packageName == null || service.packageName.isBlank()) {
+                System.err.println("Using the default java package is not supported for "
+                        + "gRPC code generation. Please specify `option java_package = \"your.package\"` in "
```

Comment:
gRPC :-D

Reply:
I would also use a logger and the "error" level.

Reply:
do you mean it should be something like `Quarkus gRPC code generation`?

Reply:
for logging, only java util logging is available there, I can switch to that. Another option is to gather all the problems and print them in the exception's message

Reply:
oh right, you are in the code generation phase. Ok, let's keep System.out for now. We may want to add some log support in the future.

Reply:
changed to JUL